### ISSUE TITLE
Text on Hover contrast

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#2405]
## Describe the bug

For the _export report_ _Start over_ _info & examples_ and _add a failure instance_ buttons, on hover we change the font color and the pointer. 
In both High contrast and default mode, the contrast for the text is not 4.5:1. After reviewing, we have agreed that text does need to have enough contrast on hover for both accessibility and usability purposes.

We are using: 
default: #0078d4
High contrast: #106ebe

## Expected behavior
default:  #0078D4 (contrast with white is 4.52:1)
High contrast: #2b88d8(contrast with black is 4.839:1)

## Screenshots
![on hover, high contrast](https://user-images.githubusercontent.com/9062104/77785900-1ca11200-701a-11ea-8e05-fd834455ae01.png)

**Comment by ferBonnin** [comment]
triaged with Peter, sounds good

**Comment by ferBonnin** [comment]
moving to needs investigation because we don't use black background we use #161616 so contrast is different

**Comment by ferBonnin** [comment]
validated in Canary
